pyproject.toml

```
@@ -32,9 +32,9 @@ packages = [{ include = "nidaqmx", from = "src" }]
 [tool.poetry.dependencies]
 python = "^3.7"
 numpy = [
-    {version="^1.20", python = "<3.10"},
-    # precompiled wheels only available in the 1.22+
-    {version="^1.22", python = "^3.10"}
+  {version=">=1.20,<1.22", python="<3.8"},
+  # precompiled wheels only available in the 1.22+
+  {version=">=1.22", python="^3.8"},
```

This comment is about Python 3.10, not 3.8.

Also, it's not really correct because 1.21.3 has precompiled wheels for Python 3.10.

```suggestion
  # numpy 1.22 dropped support for Python 3.7.
  {version=">=1.20,<1.22", python="<3.8"},
  {version=">=1.22", python="^3.8"},
```